### ISSUE TITLE
lunaryline fail from time to time

### ISSUE THREAD

**Issue by casouri** [casouri/lunarymacs#8]
Sometimes on startup lunaryline fails, switching to new buffers several times can take it back.

Reason is still unclear.

Error message:

```
Error during redisplay: (eval (let* ((active (powerline-selected-window-active)) (mode-line-buffer-id (if active (quote mode-line-buffer-id) (quote mode-line-buffer-id-inactive))) (mode-line (if active (quote mode-line) (quote mode-line-inactive))) (face0 (if active (quote powerline-active0) (quote powerline-inactive0))) (face1 (if active (quote powerline-active1) (quote powerline-inactive1))) (face2 (if active (quote powerline-active2) (quote powerline-inactive2))) (facey (if active (quote lunaryline-yellow) (quote powerline-inactive1))) (faceyl (if active (quote lunaryline-yellow-light) (quote powerline-inactive1))) (faceb (if active (quote lunaryline-blue) (quote powerline-inactive1))) (facebl (if active (quote lunaryline-blue-light) (quote powerline-inactive1))) (separator-left (intern (format "powerline-%s-%s" (powerline-current-separator) (car powerline-default-separator-dir)))) (separator-right (intern (format "powerline-%s-%s" (powerline-current-separator) (cdr powerline-default-separator-dir)))) (lhs (list (when (bound-and-true-p eyebrowse-mode) (if (buffer-modified-p) (lunaryline-eyebrowse-mode-line-indicator faceyl facey) (lunaryline-eyebrowse-mode-line-indicator facebl faceb))) (when (bound-and-true-p winum-mode) (if (buffer-modified-p) (powerline-raw (lunaryline-winum) facey (quote l)) (powerline-raw (lunaryline-winum) faceb (quote l)))) (when (bound-and-true-p winum-mode) (if (buffer-modified-p) (funcall separator-left facey face0) (funcall separator-left faceb face0))) (when powerline-display-buffer-size (powerline-buffer-size face0 (quote l))) (powerline-buffer-id (\` (mode-line-buffer-id (\, face0))) (quote l)) (powerline-raw " " face0) (funcall separator-right face0 face1) (when (and (boundp (quote erc-track-minor-mode)) erc-track-minor-mode) (powerline-raw erc-modified-channels-object face0 (quote l))) (powerline-major-mode face1 (quote l)) (powerline-raw " " face1) (only-in-prog-mode| (funcall separator-left face1 face0)) (only-in-prog-mode| (flycheck-error (quote lunaryline-flycheck-error) (quote l))) (only-in-prog-mode| (flycheck-warning (quote lunaryline-flycheck-warning) (quote l))) (only-in-prog-mode| (flycheck-info (quote lunaryline-flycheck-info) (quote l))) (only-in-prog-mode| (powerline-raw " " face0)) (only-in-prog-mode| (funcall separator-right face0 face1)) (only-in-prog-mode| (lunaryline-vc face1 (quote l))) (only-in-prog-mode| (powerline-raw " " face1)) (disappear-when-narrow| (when (bound-and-true-p nyan-mode) (funcall separator-left face1 face0))) (disappear-when-narrow| (when (bound-and-true-p nyan-mode) (powerline-raw (list (nyan-create)) face0 (quote l)))) (disappear-when-narrow| (when (bound-and-true-p nyan-mode) (funcall separator-left face0 face1))))) (rhs (list (powerline-raw global-mode-string face1 (quote r)) (funcall separator-right face1 face0) (lunaryline-encoding face0 (quote l)) (funcall separator-left face0 face1) (powerline-raw "%3l" face1 (quote l)) (powerline-raw ":" face1 (quote l)) (powerline-raw "%2c" face1 (quote r)) (funcall separator-right face1 face0) (powerline-raw " " face0) (powerline-raw "%6p" face0 (quote r)) (when powerline-display-hud (powerline-hud facey face0)) (powerline-fill face0 0)))) (concat (powerline-render lhs) (powerline-fill face1 (powerline-width rhs)) (powerline-render rhs)))) signaled (void-function s-trim) [12 times]
```

**Comment by casouri** [comment]
It should be fixed in 1d0fc7a26666293fe9a7d02f7079ce7cfb6e02e3.

**Comment by casouri** [comment]
I think the problem is my drop in copy of flycheck segment from spacemacs. I will now work on it.

**Comment by casouri** [comment]
Fixed in 602fa4d
